src/api/utils.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * @flow
+ * @file Utils for the box APIs
+ * @author Box
+ */
+
+import type { Comment } from '../common/types/feed';
+
+/**
+ * Formats comment data (including replies) for use in components.
+ *
+ * @param {Comment} comment - An individual comment entry from the API
+ * @return {Comment} Updated comment
+ */
+export const formatComment = (comment: Comment): Comment => {
+    const formattedComment = {
+        ...comment,
+        tagged_message: comment.message,
```

Comment:
Curious as to why the comments endpoint doesn't return a `tagged_message` field?

Reply:
I believe that was the decision made by API Working Group to abandon the legacy tagged_message field. Not using it in the new API for Threaded Comments was also advocated by @jstoffan.